Updated Bugs & Glitches document for the AI use of FullHeal/FullRestore not curing attack/speed debuff from paralysis or burn (#1214)
pret/pokecrystal: docs/bugs_and_glitches.md

```

### AI use of Full Heal or Full Restore does not cure confusion status

**Fix:** Edit `EnemyUsedFullRestore`, and `AI_HealStatus` in [engine/battle/ai/items.asm](https://github.com/pret/pokecrystal/blob/master/engine/battle/ai/items.asm):

```diff
 EnemyUsedFullRestore:

>  ; BUG: AI use of Full Heal or Full Restore does not cure confusion status (see docs/bugs_and_glitches.md)

Reviewer: ```suggestion
-; BUG: AI use of Full Heal or Full Restore does not cure confusion status (see docs/bugs_and_glitches.md)
```

```diff
 AI_HealStatus:

>  ; BUG: AI use of Full Heal or Full Restore does not cure Nightmare status or the attack/speed debuff from Burn/Paralysis (see docs/bugs_and_glitches.md)

Reviewer: ```suggestion
-; BUG: AI use of Full Heal or Full Restore does not cure Nightmare status (see docs/bugs_and_glitches.md)
 ; BUG: AI use of Full Heal or Full Restore does not cure Attack or Speed drops from Burn or Paralysis (see docs/bugs_and_glitches.md)
```

```diff
 AI_HealStatus:
 ; BUG: AI use of Full Heal or Full Restore does not cure Nightmare status (see docs/bugs_and_glitches.md)

>  ; BUG: AI use of Full Heal or Full Restore does not cure Attack or Speed drops from Burn or Paralysis (see docs/bugs_and_glitches.md)

Reviewer: ```suggestion
 ; BUG: AI use of Full Heal or Full Restore does not cure Attack or Speed drops from burn or paralysis (see docs/bugs_and_glitches.md)
```

 	ret
```

> ### AI use of Full Heal or Full Restore does not cure attack or speed debuffs from Burn or Paralysis

Reviewer: ```suggestion
### AI use of Full Heal or Full Restore does not cure Attack or Speed drops from Burn or Paralysis
```

**Fix:** Edit `AI_HealStatus` in [engine/battle/ai/items.asm](https://github.com/pret/pokecrystal/blob/master/engine/battle/ai/items.asm):

```diff
 AI_HealStatus:

>  ; BUG: AI use of Full Heal or Full Restore does not cure Nightmare status or the attack/speed debuff from Burn/Paralysis (see docs/bugs_and_glitches.md)

Reviewer: ```suggestion
 ; BUG: AI use of Full Heal or Full Restore does not cure Nightmare status (see docs/bugs_and_glitches.md)
-; BUG: AI use of Full Heal or Full Restore does not cure Attack or Speed drops from Burn or Paralysis (see docs/bugs_and_glitches.md)
```
